src/blockdev.rs

```diff
@@ -854,6 +849,30 @@ pub fn lsblk(dev: &Path, with_deps: bool) -> Result<Vec<HashMap<String, String>>
     Ok(result)
 }
 
+/// Parse key-value pairs from blkid.
+fn split_blkid_line(line: &str) -> HashMap<String, String> {
+    let re = Regex::new(r#"([\w/]+): (.+)"#).unwrap();
```

Comment:
Do we need a regex here or would it work fine if we just split on `:`?

Reply:
(Fine with a regex too to be clear, but I tend to avoid them if a really simple string manipulation will do.)